.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+  
+
+/samples/                       @tdh911 @googleapis/python-samples-owners
```

Comment:
@tdh911 GitHub will tag you for a review if a change is made to any of the samples. Let me know if it would be more appropriate to put someone else here.

Reply:
SGTM